.github/workflows/actions/unit-test-composite-action/action.yml

```diff
@@ -0,0 +1,32 @@
+name: unit-tests
```

Comment:
Can you move this composite action under the `.github/actions` folder? We have came up with the following pattern:

- `.github/workflows` for GH worklows
- `.github/actons` for GH reusable actions.

I'll see if we have documented this somewhere and I'll create the `dependabot` configuration to also update those github composite actions.

OTOH, I think the name `unit-test-composite-action` is unnecessary in this case; all the `action.yml` files under the `.github/actions` are composite actions in our case. Can we use a different naming? something like `go-unit-test` or `unit-test`?